components/script/dom/htmlelement.rs

```diff
@@ -560,11 +560,11 @@ static DATA_PREFIX: &str = "data-";
 static DATA_HYPHEN_SEPARATOR: char = '\x2d';
 
 fn is_ascii_uppercase(c: char) -> bool {
-    'A' <= c && c <= 'Z'
+    ('A'..='Z').contains(&c)
 }
 
 fn is_ascii_lowercase(c: char) -> bool {
-    'a' <= c && c <= 'w'
+    ('a'..='w').contains(&c)
```

Comment:
It seems weird that we use `a` to `w` as the range instead of `a` to `z`. Perhaps it was a bug introduced in [this commit](https://github.com/servo/servo/commit/4ee8f56b9cdb3c47e598eb27e94f07350fb58c5d)? It seems that commit was trying to remove the use of the standard library methods `char::is_ascii_{lower,upper}case`  because they were marked unstable at that time. But looks like those methods have been stabilized since then, so perhaps we can simply remove our definitions in this file and call the `std` methods as before?

Reply:
That is almost certainly an Azerty keyboard layout mistake.

Reply:
I have made the recommended changes.